pkg/tasks/prometheus.go

```diff
@@ -65,6 +65,21 @@ func (t *PrometheusTask) Run() error {
 		return errors.Wrap(err, "creating kubelet serving CA Bundle ConfigMap failed")
 	}
 
+	apiAuthConfigmap, err := t.client.GetConfigmap("kube-system", "extension-apiserver-authentication")
+	if err != nil {
+		return errors.Wrap(err, "failed to load kube-system/extension-apiserver-authentication configmap")
+	}
+
+	cm, err := t.factory.MetricsClientCACM(apiAuthConfigmap)
+	if err != nil {
+		return errors.Wrap(err, "initializing Metrics Client CA failed")
+	}
+
+	err = t.client.CreateOrUpdateConfigMap(cm)
+	if err != nil {
+		return errors.Wrap(err, "reconciling Metrics Client CA ConfigMap failed")
+	}
+
```

Comment:
this needs to be moved outside of the Prometheus task (probably a separate task group).

Reply:
I was wondering about that, will do!

Reply:
I added it as a new task in front of the prometheus-operator one, let's see what happens.